jdaviz/configs/cubeviz/plugins/sonify_data/sonify_data.py

```diff
@@ -0,0 +1,130 @@
+from traitlets import Bool, List, Unicode, observe
+import astropy.units as u
+
+from jdaviz.core.custom_traitlets import IntHandleEmpty, FloatHandleEmpty
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import (PluginTemplateMixin, DatasetSelectMixin,
+                                        SpectralSubsetSelectMixin, with_spinner)
+from jdaviz.core.user_api import PluginUserApi
+
+
+__all__ = ['SonifyData']
+
+try:
+    import strauss  # noqa
+    import sounddevice as sd
+except ImportError:
+    class Empty:
+        pass
+    sd = Empty()
+    sd.default = Empty()
+    sd.default.device = [-1, -1]
+    _has_strauss = False
+else:
+    _has_strauss = True
+
+
+@tray_registry('cubeviz-sonify-data', label="Sonify Data",
+               viewer_requirements=['spectrum', 'image'])
+class SonifyData(PluginTemplateMixin, DatasetSelectMixin, SpectralSubsetSelectMixin):
+    """
+    See the :ref:`Sonify Data Plugin Documentation <cubeviz-sonify-data>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.close_in_tray`
+    """
+    template_file = __file__, "sonify_data.vue"
+
+    sample_rate = IntHandleEmpty(44100).tag(sync=True)
+    buffer_size = IntHandleEmpty(2048).tag(sync=True)
+    assidx = FloatHandleEmpty(2.5).tag(sync=True)
+    ssvidx = FloatHandleEmpty(0.65).tag(sync=True)
+    eln = Bool(False).tag(sync=True)
+    audfrqmin = FloatHandleEmpty(50).tag(sync=True)
+    audfrqmax = FloatHandleEmpty(1500).tag(sync=True)
+    pccut = IntHandleEmpty(20).tag(sync=True)
+    volume = IntHandleEmpty(100).tag(sync=True)
+    stream_active = Bool(True).tag(sync=True)
+    has_strauss = Bool(_has_strauss).tag(sync=True)
+
+    # TODO: can we referesh the list, so sounddevices are up-to-date when dropdown clicked?
+    sound_devices_items = List().tag(sync=True)
+    sound_devices_selected = Unicode('').tag(sync=True)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._plugin_description = 'Sonify a data cube'
+        self.docs_description = 'Sonify a data cube using the Strauss package.'
+        if not self.has_strauss or sd.default.device[1] < 0:
+            self.disabled_msg = ('To use Sonify Data, install strauss and restart Jdaviz. You '
+                                 'can do this by running `pip install ".[strauss]"` in the command'
```

Comment:
should this be updated to `pip install strauss` now?  Back ticks also won't render as code this way... maybe better without them entirely?

Reply:
Related, I realized that my [comment here](https://github.com/spacetelescope/jdaviz/pull/3269#discussion_r1888914657) was never addressed. Right now the instructions are confusing because they don't say that this command has to be run in the Jdaviz root directory, and also aren't applicable if the user is not on a dev install.

Reply:
The correct syntax is `pip install jdaviz[strauss]` for end-users.

ref: https://github.com/astropy/astropy/wiki/v7.0.0-RC-testing

Reply:
Now that strauss is released, can we just suggest `pip install strauss` or do we want to maintain the ability to pin specific version by suggesting `[strauss]`?

Reply:
I'll change it in both places to `pip  install strauss`.